plugin/src/queries.ts

```diff
@@ -132,6 +132,18 @@ const collectionsQuery = (dateString?: string) => `
         templateSuffix
         title
         updatedAt
+        metafields {
+          edges {
+            node {
+              description
+              id
+              key
+              namespace
+              value
+              valueType
+            }
+          }
+        }
```

Comment:
Now that I think of it, I suspect the reason why this just works without updating the processor logic is because it appears in the query after the products, and it therefore the metafields entries also appear in the results after the products for the same collection. So we process the metafields naively, and then we process the products with the custom logic.

If, on the other hand, the `metafields` were to appear before `products` in this query, I think the nodes would not get created. My hypothesis anyway. And if that's true, it means that this works accidentally, which might be okay for now as long as I follow up on it.